packages/kuma-gui/src/app/connections/views/ConnectionOutboundSummaryClustersView.vue

```diff
@@ -9,48 +9,56 @@
+    v-slot="{ route, uri }"
   >
     <RouteTitle
       :render="false"
       :title="`Clusters`"
     />
     <AppView>
       <DataLoader
-        :src="`/meshes/${route.params.mesh}/dataplanes/${route.params.dataPlane}/data-path/clusters`"
-        v-slot="{ data, refresh }: ClustersDataSource"
+        :src="uri(sources, '/meshes/:mesh/dataplanes/:name/clusters', {
+          mesh: route.params.mesh,
+          name: route.params.dataPlane,
+        })"
+        v-slot="{ data, refresh }"
       >
-        <DataCollection
-          :items="data!.split('\n')"
-          :predicate="item => item.startsWith(`${route.params.connection}::`)"
-          v-slot="{ items: lines }"
+        <template
+          v-for="prefix in [route.params.connection]"
+          :key="typeof prefix"
         >
-          <XCodeBlock
-            language="json"
-            :code="lines.map(item => item.replace(`${route.params.connection}::`, '')).join('\n')"
-            is-searchable
-            :query="route.params.codeSearch"
-            :is-filter-mode="route.params.codeFilter"
-            :is-reg-exp-mode="route.params.codeRegExp"
-            @query-change="route.update({ codeSearch: $event })"
-            @filter-mode-change="route.update({ codeFilter: $event })"
-            @reg-exp-mode-change="route.update({ codeRegExp: $event })"
+          <DataCollection
+            :items="data.split('\n')"
+            :predicate="item => item.startsWith(`${prefix}::`)"
+            v-slot="{ items: lines }"
           >
-            <template #primary-actions>
-              <XAction
-                action="refresh"
-                appearance="primary"
-                @click="refresh"
-              >
-                Refresh
-              </XAction>
-            </template>
-          </XCodeBlock>
-        </DataCollection>
+            <XCodeBlock
+              language="json"
+              :code="lines.map(item => item.replace(`${prefix}::`, '')).join('\n')"
+              is-searchable
+              :query="route.params.codeSearch"
+              :is-filter-mode="route.params.codeFilter"
+              :is-reg-exp-mode="route.params.codeRegExp"
+              @query-change="route.update({ codeSearch: $event })"
+              @filter-mode-change="route.update({ codeFilter: $event })"
+              @reg-exp-mode-change="route.update({ codeRegExp: $event })"
+            >
+              <template #primary-actions>
+                <XAction
+                  action="refresh"
+                  appearance="primary"
+                  @click="refresh"
+                >
+                  Refresh
+                </XAction>
+              </template>
+            </XCodeBlock>
+          </DataCollection>
+        </template>
       </DataLoader>
     </AppView>
   </RouteView>
 </template>
 <script lang="ts" setup>
-import type { ClustersDataSource } from '@/app/data-planes/sources'
+import { sources } from '@/app/data-planes/sources'
```

Comment:
The changes in this file are just to make it as consistent as possible with inbounds, the only things that changes is the setting of the `prefix` property.